### ISSUE TITLE
learning rate 3e-4 --> 3e-5 gives better results (as original paper)

### ISSUE THREAD

**Issue by yxxshin** [aim-uofa/StyleDrop-PyTorch#9]
I didn't dive deep for this project, but I'm curious why the model uses lr = 0.0003 ( at `custom.py` )
The original paper introduces lr = 0.00003. I found that 1000 steps with lr = 3e-4 gives a highly overfitted result, while lr = 3e-5 was much better (though both quality is not really good)

**Comment by zideliu** [comment]
Since our basic models are different, the parameters in the paper are not suitable for this reimplement version. In addition, I set the parameters based on the final training of this base model at e-4 level.

**Comment by yxxshin** [comment]
Ah, got it. Thanks for the wonderful work!
